scripts/prepush.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+protected_branch1='master'
+protected_branch2='stable'
+current_branch=$(git symbolic-ref HEAD | sed -e 's,.*/\(.*\),\1,')
+
+if [ $protected_branch1 = $current_branch ]
```

Comment:
Can't we just change this to use an OR on the two branches 
`if branch is stable or master`

Reply:
Yeah we probably can, my bash is just shit so I took the easy way out and copied the script I found online twice instead of doing it properly :P. I'll look at it haha